recipe/meta.yaml

```diff
@@ -10,35 +10,44 @@ source:
   sha256: 1bf89ae58050bbd32c7307199046117feee245c2fd9ab6255c7308522b7ca149
 
 build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
   entry_points:
     - hyperopt-mongo-worker=hyperopt.mongoexp:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
 
 requirements:
   host:
+    - python
+    - numpy
```

Comment:
Please add here the '={{ numpy }}' constrain to it

Reply:
Done